pkg/web3/api.go

```diff
@@ -90,8 +90,7 @@ func (sdk *Web3SDK) GetSolverUrl(address string) (string, error) {
 		common.HexToAddress(address),
 	)
 	if err != nil {
-		log.Error().Msgf("GetUser error")
-		log.Error().Msgf("error: %s", err)
+		log.Error().Msgf("Failed to discover to solver URL: %s", err)
```

Comment:
[nit] small thing but is there an extra `to` between discover and solver?

Reply:
Ah yeah, good catch 💯. Fixed it.